core/controllers/admin.py

```diff
@@ -52,6 +80,31 @@ def test_super_admin(self, **kwargs):
     return test_super_admin
 
 
+def check_and_update_config_role(old_list, new_list, role):
+    """Changes roles for users in old_list and new_list. If user belongs to
+    new_list but not in old list, its role is changed to specified role. If a
+    user belongs to old list but not new list, its role is changed to
+    exploration editor.
+
+    Args:
+        old_list: list(str). Existing list of usernames with given
+            role.
+        new_list: list(str). Updated list of usernames with given role.
+        role: str. The role to be used for users in these lists.
+    """
+    for username in old_list:
```

Comment:
uh, oh. I didn't know a user can belong to multiple lists. I have a few points:
- This is to keep sync after the migration job is run till the new system replaces old one.
- The one off job directly maps user roles from lists to the roles (eg. all users in admin_usernames list get role 'admin', all users in moderator_usernames get role 'moderator' and so on).
- If while running migration admin_usernames, moderator_usernames etc lists would be modified so as to uniquely map in terms of the new system, would they be reverted back to having usernames in multiple lists?